File: C/Listas/Lista 02/ex09.c
//Faça um programa que leia as variáveis y e z. Após, calcule e mostre o valor de x, dado pela expressão: x =  y*(y^22 + z^33)
#include <stdio.h>
#include <math.h>

int main(){
    float x,y,z,interno,poty,potz;
    printf("Informe, respectivamente, um valor para y e para z: ");
    scanf("%f%f", &y,&z);
    poty=pow(y,22);
    potz=pow(z,33);
    interno=poty+potz;
    x=sqrt(interno);
    
    printf("A raiz quadrada de %.0f será: %.0f",interno, x);
    return 0;
}

File: Java/extra/Lista 02/src/ex09.java
/*  Faça um programa que leia as variáveis y e z. Após, calcule e mostre o valor de x, 
dado pela expressão:
x =  y*(y22 + z33) */

import java.util.Scanner;

public class ex09 {
    static void raiz(int a, int b) {
        double x = Math.sqrt(a * (Math.pow(a, 22) + Math.pow(b, 33)));

        System.out.printf("\n%.2f", x);
    }

    public static void main(String[] args) {
        int n1, n2;
        System.out.print("Informe o valor de Y: ");
        try (Scanner leitor = new Scanner(System.in)) {
            n1 = leitor.nextInt();
            System.out.print("Informe o valor de Z: ");
            n2 = leitor.nextInt();
        }
        raiz(n1, n2);
    }
}